.github/workflows/tox.yml

```
@@ -22,12 +22,6 @@ jobs:
         include:
           - tox_env: lint
           # - tox_env: docs
-          - tox_env: py36
-            PREFIX: PYTEST_REQPASS=2
-          - tox_env: py36-devel
-            PREFIX: PYTEST_REQPASS=2
-          - tox_env: py37
-            PREFIX: PYTEST_REQPASS=2
           - tox_env: py38
             PREFIX: PYTEST_REQPASS=2
```

might be a good idea to move the first `devel` target and create an additional `py38-devel`.
```suggestion
          - tox_env: py38
            PREFIX: PYTEST_REQPASS=2
          - tox_env: py38-devel
            PREFIX: PYTEST_REQPASS=2
```